src/modules/users/useCases/createMessage/CreateMessageController.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { app } from "@shared/infra/http/app";
+import { IUserForTest, user } from "@shared/infra/jest/jest.setup";
+import { randomUUID } from "crypto";
+import request from "supertest";
+
+describe("CreateMessageController", () => {
+  const user2: IUserForTest = {
+    full_name: "John Doe",
```

Comment:
John Doe is famous in ur country? Every file i saw that guy 😢

Reply:
He is a good man, everyone knows about him...